.github/workflows/test.yml

```diff
@@ -7,6 +7,30 @@ on:
     branches: [ 'master' ]
 
 jobs:
+  black-linting:
+    runs-on: ubuntu-latest
+    name: Black Lint
+    steps:
+      - uses: actions/checkout@v2
+      - uses: psf/black@stable
+        with:
+          options: "--check --verbose"
+          src: "./python"
+  flake8-lint:
+    runs-on: ubuntu-latest
+    name: Flake8 Lint
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v2
+      - name: Set up Python environment
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+      - name: flake8 Lint
+        uses: py-actions/flake8@v2
+        with:
+          args: "--config python/.flake8"
+          path: "./python"
   run:
```

Comment:
Now that we have multiple jobs in this workflow, let's give this one a more descriptive name than just "run". Maybe "test" and `name: Build and Test Module`